apple/sensor/ReanimatedSensor.m

```diff
@@ -1,6 +1,6 @@
 #import <RNReanimated/ReanimatedSensor.h>
 
-#if !TARGET_OS_TV && !TARGET_OS_OSX
+#if !TARGET_OS_TV && !TARGET_OS_OSX && !TARGET_OS_VISION
 @implementation ReanimatedSensor
```

Comment:
![CleanShot 2024-02-02 at 11 59 08@2x](https://github.com/software-mansion/react-native-reanimated/assets/52801365/7ead4794-df03-439c-b8b7-e0efbc48340c)

This could be an interesting extension 😄 I'll get back to this module once I have real device and try to make it work for visionOS